lib/cb/clients/base.rb

```diff
@@ -14,6 +14,14 @@ class Base
       def self.cb_client
         @cb_client ||= Cb::Utils::Api.instance
       end
+
+      def self.headers(args)
+        {
+            'Accept' => 'application/json',
+            'Authorization' => "Bearer #{ args[:oauth_token] }",
+            'Content-Type' => 'application/json'
+        }
+      end
```

Comment:
Are you having to use a different oauth token than what c-m would provide? Or is this so that you just don't have to use the token.get ?